### DISCUSSION TITLE
FR: Log failed login attempts for the use of Fail2Ban

### DISCUSSION THREAD

**Discussion by josvink66** [mealie-recipes/mealie#2346] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [X] Reviewed the documentation and verified that my feature request isn't already a feature.
- [X] Reviewed the existing feature requests and my feature has not been requested.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
I have Mealie running behind SWAG. SWAG has build in Fail2Ban system. But to use this feature, we need the (failed) login logs.

**(Optional): Suggest A Solution**
Log failed login attempts to a file so we can use Fail2Ban regex to locate and ban failed login attempts

**Additional Information**
No additional information

**Comment by jecorn** [comment]
This should be pretty easy to implement. I'm a bit underwater at work at the moment, but could look into it time next month. The problem is that I'm not reverse proxying mealie, so it's difficult for me to test whether external IPs are getting properly detected through the reverse proxy. It would not be great to fail2ban the proxy itself. @josvink66 would you be willing to pull from a development branch on my fork to test it out?

**Comment by jecorn** [comment]
FYI, this ended up being a 5-liner, so the pull request is already in. I was able to test IP address reporting on my local network without a reverse proxy, but I don't have a reverse proxy of my own to test with. Correct passing along of client IP through the reverse proxy is (I think) solely up to user config of the reverse proxy.
https://github.com/hay-kot/mealie/pull/2365
